bin/drug-screening-workflow/dose-response-fitting-tool.cwl

```diff
@@ -0,0 +1,32 @@
+class: CommandLineTool
+cwlVersion: v1.0
+$namespaces:
+  sbg: 'https://www.sevenbridges.com/'
```

Comment:
what does this mean?

Reply:
this is a rabix artifact as well, has to do with the coords i think.